.github/ISSUE_TEMPLATE/04 build_failure.yml

```
@@ -0,0 +1,162 @@
+name: "Build Failure"
```

```suggestion
name: "Build failure"
```
The [current capitalization strategy](https://github.com/NixOS/nixpkgs/issues/new/choose) is easier to read since the first word carries more weight. Should the same strategy be used for this PR?

---

Yes, it should. That was my bad.

---

FYI I am keeping the `Request: something` format from Pluie's review, but only `Request` will be capitalised now.